README.md

```diff
@@ -0,0 +1,26 @@
+## AI Tutor XBlock
```

Comment:
Can we add little description of what this xblock does and possible use cases? Also we should add few screenshots of xblock(in studio and LMS) like [this](https://github.com/edly-io/h5pxblock#readme)